scripts/log-session.py

```
@@ -0,0 +1,334 @@
+        if (cwd / 'CLAUDE.md').exists():
+            memory_dir = cwd / 'memory'
+            memory_dir.mkdir(exist_ok=True)
+            return memory_dir
+        cwd = cwd.parent
+    raise FileNotFoundError("Could not find project root (CLAUDE.md)")
+
+
+def get_today_log() -> Path:
+    """Get path to today's log file."""
+    memory_dir = get_memory_dir()
+    today = datetime.now().strftime('%Y-%m-%d')
```

Ruff (0.14.14)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)